tests/manifests/special-cases/no-strategy/controller-config.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: numaplane-controller-config
 data:
   config.yaml: |
-    logLevel: INFO # Supported log levels are: VERBOSE, DEBUG, INFO, WARN. ERROR, FATAL will be printed regardless of the log level
+    logLevel: DEBUG # Supported log levels are: VERBOSE, DEBUG, INFO, WARN. ERROR, FATAL will be printed regardless of the log level
```

Comment:
I'm not sure why we had this set to INFO - I'm guessing we generally didn't look at this test

Reply:
Probably, As it was set as `3` earlier, So respectively replaced with `INFO`